src/components/About/About.js

```diff
@@ -80,9 +80,12 @@ const About = (props) => {
       default:
         headlineMsg = <FormattedMessage id="stripes-core.about.moduleTypeCount" values={{ count: list.length, type: caption }} />;
     }
+
+    list.sort();
+
     return (
       <div key={caption}>
-        <Headline>{headlineMsg}</Headline>
+        <Headline>headline: {headlineMsg}</Headline>
```

Comment:
Should this be localized in `en.json`?

Reply:
Why was this text added at all?

Reply:
That was debugging; it should be removed. Good catch, guys; thank you.